src/renderer/viz/expressions/buckets.js

```diff
@@ -4,60 +4,49 @@ import { implicitCast, getOrdinalFromIndex } from './utils';
 /**
  * Given a property create "sub-groups" based on the given breakpoints.
  *
- * Imagine a traffic dataset with a speed property. We want to divide the roads in
- * 3 buckets (slow, medium, fast) based on the speed using a different color each bucket.
+ * This returns a number or category expression depending on the input values.
  *
- * We´ll need:
- *  - A {@link carto.expressions.ramp|ramp} to add a color for every bucket.
- *  - A {@link carto.expressions.palettes|colorPalette} to define de color scheme.
+ * @param {Number|Category} property - The property to be evaluated and interpolated
+ * @param {Number[]|Category[]} breakpoints - Expression containing the different breakpoints.
+ * @return {Number|Category}
  *
- * ```javascript
- *  const s = carto.expressions;
- *  const $speed = s.prop('speed');
- *  const viz = new carto.Viz({
+ * @example <caption>Display a traffic dataset is 3 colors depending on the numeric speed.</caption>
+ * // Using the buckets `expression` we divide the dataset into 3 buckets according to the speed
+ * // - From 0 to 29
+ * // - From 30 to 79
+ * // - From 80 to 120
+ * // Values lower than 0 will be in the first bucket and values higher than 120 will be in the third one.
+ * const s = carto.expressions;
+ * const viz = new carto.Viz({
  *    color: s.ramp(
- *      s.buckets($speed, [30, 80, 120]),
+ *      s.buckets(s.prop('speed'), [30, 80, 120]),
  *      s.palettes.PRISM
  *    )
  * });
- * ```
  *
- * ```javascript
- *  const viz = new carto.Viz(`
+ * @example <caption>Display a traffic dataset is 3 colors depending on the numeric speed. (String)</caption>
+ * // Using the buckets `expression` we divide the dataset into 3 buckets according to the speed
+ * // - From 0 to 29
+ * // - From 30 to 79
+ * // - From 80 to 120
+ * // Values lower than 0 will be in the first bucket and values higher than 120 will be in the third one.
```

Comment:
the error I was referring is this `third`, it should be `last` or `fourth`

Reply:
Sorry but I don't get why `third` wrong. The buckets are (-inf, 29], [30, 79], [80, inf) right?

Reply:
Nop, the buckets are (-inf, 29], [30, 79], [80, 120), [120, inf)

Reply:
Oh, didn't know that. So the clarification is required.

Reply:
(Updated)